### PR TITLE
Remove university separation

### PR DESCRIPTION
**Pull request by acknowledge**
It simply remove the university separation and order people by alphabetical order, as discussed. The two new members were added to the list with a placeholder profile picture.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -165,39 +245,36 @@
...
-
-        <h1>ETHZ</h1>
-        <div class="container">
             <div class="person-container">
                 <div class="person">
                     <img class="photo"
-                         src="images/people/xandeep_alexander.jpg"
-                         alt="Picture of Xandeep Alexander">
+                         src="images/people/placeholder.jpg"
+                         alt="Picture of Tiana Nikitina">
```
Comment (SolalPirelli) [inline index.html]: Tiana is a short name, write "Tatiana" in full for a website like this :)

Comment (acknowledge) [inline index.html]: Oh ok, no problem. It's done.

File: index.html [hunk 2]
```diff
@@ -23,12 +23,13 @@
...
+                Welcome to the MSP Switzerland page.
             </p>
             <p>
-                Second paragraph of text, lalalalala... probably need a <a href="about:blank"
-                                                                           rel="noreferrer"
-                                                                           target="_blank">link</a> to apply or something...
+                We're a bunch of students passionate about technology from all over Switzerland. Our goal is to promote Microsoft products from HoloLens, Xbox, Kinect and Surface to services like Azure, Cognitive Services or Office across Switzerland. To achieve this goal we organize events such as hackathons and talks.
+            </p>
+            <p>
+                Visit our <a href="https://www.facebook.com/mspswitzerland" rel="noreferrer" target="_blank">facebook page</a> for more details, pics, infos.
```
Comment (SolalPirelli) [inline index.html]: Capital F on Facebook? :D

Comment (acknowledge) [inline index.html]: It already is !

Comment (acknowledge) [inline index.html]: I don't know why commits from Sep 18 are in this PR. But this was already resolved.

**Pull request merged**